src/cljs/rems/actions/request_decision.cljs

```diff
@@ -57,39 +32,28 @@
                   :on-click #(rf/dispatch [::open-form])}])
 
 (defn request-decision-view
-  [{:keys [application-id selected-deciders potential-deciders on-set-deciders on-send]}]
+  [{:keys [application-id disabled on-send]}]
   [action-form-view action-form-id
    (text :t.actions/request-decision)
    [[button-wrapper {:id "request-decision"
                      :text (text :t.actions/request-decision)
                      :class "btn-primary"
                      :on-click on-send
-                     :disabled (empty? selected-deciders)}]]
+                     :disabled disabled}]]
    [:div
     [comment-field {:field-key action-form-id
                     :label (text :t.form/add-comments-not-shown-to-applicant)}]
     [action-attachment {:field-key action-form-id
                         :application-id application-id}]
-    [:div.form-group
-     [:label {:for dropdown-id} (text :t.actions/request-selection)]
-     [dropdown/dropdown
-      {:id dropdown-id
-       :items potential-deciders
-       :item-key :userid
-       :item-label :display
-       :item-selected? #(contains? (set selected-deciders) %)
-       :multi? true
```

Comment:
See here, decider selection has had `multi? true`. The event & command also support multiple users.

Reply:
Well it could potentially be wrong. We don't have it as configurable and I guess that's ok for now, but the assumption we had in an earlier discussion couple weeks ago (when doing the wording change) was that there would only ever be one decider.

Reply:
Which wording change? Looking at the history, the label has been `:t.actions/request-selection` and `multi?` has been `true` at least since 2019-08-01, commit a6c15aae1c5afcbeb66706434b0b30f36a6a646b.

Reply:
https://github.com/CSCfi/rems/pull/2397

Reply:
Right! What do you propose? I find that the UI is clear enough even with multiselection, and it doesn't really do any harm to support multiple deciders.

Reply:
Well I would just like to confirm from @jaakkocsc  that it's so far ok to have multiple deciders. The changes look fine otherwise.

Reply:
I'd say the decision on having only one decider or multiple deciders is organisational, by the "organisation-owner", not a technical limitation set by REMS.

Reply:
It simplifies REMS to have all the invites/requests to be available for multiple people.